NuRadioMC/utilities/attenuation.py

```diff
@@ -71,5 +71,11 @@ def get_attenuation_length(z, frequency, model):
             att_length_f[ att_length_f < min_length ] = min_length
 
         return att_length_f
+    elif(model == "MB1"): # from 10.3189/2015JoG14J214
```

Comment:
@cg-laser I think the MB model would have been nicer in a new branch, but I can live with this.

Reply:
I agree but I didn't know how to do it differently at the moment because we can only merge pull requests into the master branch (at least through the web interface).